libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -54,7 +55,18 @@ func loadBridgeNetFilterModule(fullPath string) error {
 }
 
 // Enable bridge net filtering if not already enabled. See GitHub issue #11404
-func enableBridgeNetFiltering(nfParam string) error {
+func enableBridgeNetFiltering(nfParam string) (retErr error) {
+	defer func() {
+		if retErr != nil {
+			if ignore, err := strconv.ParseBool(os.Getenv("DOCKER_IGNORE_BR_NETFILTER_ERROR")); err == nil && ignore {
+				log.G(context.TODO()).WithError(retErr).Warnf("Continuing without enabling br_netfilter")
+				retErr = nil
+				return
+			}
+			retErr = fmt.Errorf("%w: DOCKER_IGNORE_BR_NETFILTER_ERROR=1 to ignore", retErr)
```

Comment:
Given that we document  it like this, I guess we can even just check for;

```go
if os.Getenv("DOCKER_IGNORE_BR_NETFILTER_ERROR") == "1" {
```

(but just "any non-empty value" would still work))